### PR TITLE
assignment 4

### PR DESCRIPTION
**Pull request by Formil**

### CODE REVIEW

**Comment by dgarijo** [comment]
@Formil please separate assignment 4 and 2 in two different pull requests. You may do this using branches.

File: Assignment4/Formil-b190422/task06.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+for s,p,o in g.triples((ns.JaneSmith, None, None)):
+  print(s,p,o)
+print("------------------------------------------------------------------------------------------------------------------------------------------")
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+print("Task 6.5")
+ns = Namespace("http://somewhere#")
+rdfs = Namespace("https://www.w3.org/TR/rdf-schema/#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.UPM,RDF.type,ns.University))
+g.add((ns.JaneSmith,ns.WorksAt,ns.UPM))
```
Comment (dgarijo) [inline Assignment4/Formil-b190422/task06.py:82]: you have never added worksAt as a property. You should add it first to your graph

File: Assignment4/Formil-b190422/task06.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+# Visualize the results
+print("Printeamos los triples que tengan que ver con Jane Smith")
+for s,p,o in g.triples((ns.JaneSmith, None, None)):
+  print(s,p,o)
+print("------------------------------------------------------------------------------------------------------------------------------------------")
```
Comment (dgarijo) [inline Assignment4/Formil-b190422/task06.py:87]: You are missing this year's assignment

File: Assignment4/Formil-b190422/task07.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+rdfs = Namespace("http://www.w3.org/2000/01/rdf-schema#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+print("Printeamos los resultados obtenidos con rdfslib")
+for s,p,o in g.triples((None,rdfs.subClassOf,ns.Person)):
+  print(s)
+# Visualize the results
+
+q1 = prepareQuery('''
+  SELECT ?Subject WHERE { 
+    ?Subject rdfs:subClassOf ns:Person
```
Comment (dgarijo) [inline Assignment4/Formil-b190422/task07.py:38]: missing all other subclasses of person. Also, you are copying this from last year. This is not this year's assignment. I will stop my comments here.

**Review by dgarijo — CHANGES_REQUESTED** [review]
You are not doing the assignment from this year. It's different from last year.

**Comment by Formil** [comment]
Sorry, uploaded wrong files, will upload the correct ones as soon as i get home today and without incorrect commits, thanks for notifying

**Pull request closed without merging**